### DISCUSSION TITLE
ActionIcon - Lobe UI

### DISCUSSION THREAD

**Discussion by giscus** [lobehub/lobe-ui#31] [Q&A]
# ActionIcon - Lobe UI

ActionIcon is a component used to render an icon with a block around it

https://ui.lobehub.com/components/action-icon

**Comment by 1339510704** [comment]
新建一个电商平台页面

**Comment by laurturcu** [comment]
asdasdasd

**Comment by zhangxin0723** [comment]
文档真简陋 用起来全靠着摸索
